### ISSUE TITLE
android

### ISSUE THREAD

**Issue by ekaggrat** [dronus/VisPrinter#2]
is it possible to run this on a android tablet/phone with bluetooth?

**Comment by doryosef** [comment]
No
בתאריך 16 באפר 2014 05:02, "ekaggrat" notifications@github.com כתב:

**Comment by dronus** [comment]
I don't think it would be of much use of running the VisPrinter server on an android device. It would be more useful to attach a RaspberryPi for example to the printer and run the server there. 

It is possible to access the interface with an android device by the network (eg. WiFi) then, however there are not many browsers supporting the WebGL for 3d preview on android. But Opera and newer Firefox browsers on Android may work.

VisPrinter is meant to provide access to the 3d printer by all devices that run a browser, and that are many...

Maybe you like to check out other peoples forks of VisPrinter, that may be better suited to mobile browsers.
